src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs

```diff
@@ -0,0 +1,320 @@
+            {
+                // Arrange the data before starting the services
+                string mapServicePath = await GetMpkPath();
+                string featureServicePath = await GetFeatureLayerPath();
+                string geoprocessingPath = await GetGpPath();
+
+                // Create each service but don't start any 
+                _localMapService = new LocalMapService(mapServicePath);
+                _localFeatureService = new LocalFeatureService(featureServicePath);
+                _localGeoprocessingService = new LocalGeoprocessingService(geoprocessingPath);
+
+                // Subscribe to status updates for each service
+                _localMapService.StatusChanged += (o, e) => { UpdateUiWithServiceUpdate("Map Service", e.Status); };
+                _localFeatureService.StatusChanged += (o, e) => { UpdateUiWithServiceUpdate("Feature Service", e.Status); };
+                _localGeoprocessingService.StatusChanged += (o, e) => { UpdateUiWithServiceUpdate("Geoprocessing Service", e.Status); };
+
+                // Enable the UI to select services
+                comboServiceSelect.IsEnabled = true;
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, "Failed to create services");
+            }
+        }
+
+        
+
+        private void Selector_OnSelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            // Get the text of the selected item
+            string selection = ((ComboBoxItem)comboServiceSelect.SelectedItem).Content.ToString();
+
+            // Update the selection
+            switch (selection)
+            {
+                case "Map Service":
+                    _selectedService = _localMapService;
+                    break;
+
+                case "Feature Service":
+                    _selectedService = _localFeatureService;
+                    break;
+
+                case "Geoprocessing Service":
+                    _selectedService = _localGeoprocessingService;
+                    break;
+            }
+
+            // Return if selection is invalid
+            if (_selectedService == null)
+            {
+                return;
+            }
+
+            // Update the state of the start and stop buttons
+            UpdateServiceControlUi();
+        }
+
+        private void UpdateServiceControlUi()
+        {
+            if (_selectedService == null)
+            {
+                return;
+            }
+
+            // Update the UI
+            if (_selectedService.Status == LocalServerStatus.Started)
+            {
+                btnServiceStop.IsEnabled = true;
+                btnServiceStart.IsEnabled = false;
+            }
+            else
+            {
+                btnServiceStop.IsEnabled = false;
+                btnServiceStart.IsEnabled = true;
+            }
+        }
+
+        private async void BtnStartServiceClick(object sender, RoutedEventArgs e)
```

Comment:
I don't like the Hungarian notation naming of controls <yuck>. That's just my opinion ;).